client_lib/pulp/client/launcher.py

```diff
@@ -192,7 +192,7 @@ def _create_bindings(config, cli_logger, username, password, verbose=None):
         cert_filename = None
 
     api_logger = None
-    if verbose and verbose > 1:
+    if verbose and verbose >= 1:
```

Comment:
It was my understanding that we only wanted to log API output with -vv and not with just -v.